volunteer.markdown

```diff
@@ -13,24 +13,27 @@ layout: page
 ## Get in touch and talk to us
 Join the conversation on [Slack](http://slack.hotosm.org/)! A large number of volunteers and staff discuss mapping, emergencies, Working Groups and other matters on our Slack channel.
 
-## Mapping Volunteers
+<span class="anchor" id="mapping-volunteers"></span>
+<h2>Mapping Volunteers</h2>
 
 As a mapping volunteer, you can get involved with projects around the world, and contribute to humanitarian mapping through OpenStreetMap. Even if you are a new mapper, anyone can use a computer to create basemaps by drawing over buildings and roads, or use a phone to swipe and tap population hotspots in the MapSwipe app. If you have never mapped before and would like to help, visit the Tasking Manager's [Learn page](https://tasks.hotosm.org/learn) to get started mapping in less than 30 minutes.
 
 <a href="https://tasks.hotosm.org/learn" class="btn btn-primary btn-lg btn-chevron">Learn how to map</a>
 
 Please also join the discussion on Slack (see above).
 
-## Humanitarian and GIS Professionals
+<span class="anchor" id="humanitarian-and-gis-professionals"></span>
+<h2>Humanitarian and GIS Professionals</h2>
 
 Are you a humanitarian or GIS professional? In addition to remote mapping, you can volunteer with HOT in other mapping related tasks. We could use help on data processing, validating remote mapping, or creating maps and other visualizations. 
 
 We also have opportunities to support field projects, both volunteer (remote) support and in the field. Job and volunteering opportunities will also be posted on the [Jobs page](http://www-dev.hotosm.org/jobs/).
 
-## Software Engineers
+<span class="anchor" id="software-engineers"></span>
```

Comment:
Please move the HTML to Please move this to https://github.com/hotosm/hotosm-website/blob/gh-pages/_layouts/get-involved.html#L18-L20.